src/alerts/version-polling.test.ts

```diff
@@ -44,16 +44,21 @@ describe('checkVersion', () => {
       expect(getBadVersions).toHaveBeenCalled();
     });
 
-    it('sets updated required flag if current version is bad', async () => {
-      // Arrange
-      asMockedFn(getBadVersions).mockResolvedValue(['abcd123']);
-
-      // Act
-      await checkVersion();
-
-      // Assert
-      expect(versionStore.get()).toMatchObject({ isUpdateRequired: true });
-    });
+    it(
+      'sets update required time if current version is bad',
+      withFakeTimers(async () => {
+        // Arrange
+        jest.setSystemTime(1702396702141);
+        asMockedFn(getBadVersions).mockResolvedValue(['abcd123']);
+
+        // Act
+        await checkVersion();
+
+        // Assert
+        const { updateRequiredBy } = versionStore.get();
+        expect(updateRequiredBy).toBe(1702396822141);
```

Comment:
How come this doesn't have the delay added on?

Reply:
Done in https://github.com/DataBiosphere/terra-ui/pull/4559/commits/e08607085b00acbdffa8ea1015f0c568570d8627.